determined-ai/determined
# chore: replace modal hooks with UIKit modals in model and model version pages

## Description
[WEB-1103]




## Test Plan
- Check if all modals from model and model version work the same as before
- Check if tags and metadata validations work in `create model modal`
  - No duplicate keys or tags
  - No empty key or tags




## Commentary (optional)
- The behavior of `create model modal` in checkpoint modal is changed due to the difficulty of migration with checkpoint modal
  - It used to open checkpoint modal again after creating model, but it doesn't with new `create model modal` since we don't use context holder anymore.  I think this should be able to fix it in checkpoint modal migration. I added `TODO` comment in each checkpoint modal usage




## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1103]: https://hpe-aiatscale.atlassian.net/browse/WEB-1103?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
